src/components/CandidateForm.jsx

```diff
@@ -25,8 +38,13 @@ const CandidateForm = ({ fields, onSubmit }) => {
     // 3. copy values
     // find index of technologies input
     // loop through the selected values
+    const newValues = values.concat({
```

Comment:
This will only work the first time when there is no technologies field in the `values`. If you would like to edit the technologies field the second time, concat would just add a new technologies object to the `values` array, not merge it with the existing one. 

It would be better if we would have an empty `technologies` object in the `fields` passed to `CandidateForm` from the start. And then we would just find that field and change it.